utilities/transactions/pessimistic_transaction_db.h

```diff
@@ -232,11 +245,11 @@ class WritePreparedTxnDB : public PessimisticTransactionDB {
   // A heap of prepared transactions. Thread-safety is provided with
   // prepared_mutex_.
   PreparedHeap prepared_txns_;
-  // 10m entry, 80MB size
-  static const uint64_t COMMIT_CACHE_SIZE = static_cast<uint64_t>(1 << 21);
-  // commit_cache_ is initialized to zero to tell apart an empty index from a
-  // filled one. Thread-safety is provided with commit_cache_mutex_.
-  CommitEntry commit_cache_[COMMIT_CACHE_SIZE] = {};
+  static uint64_t DEF_COMMIT_CACHE_SIZE;
```

Comment:
You can have TransactionDB::WrapDB pass in the param to the constructor without adding new options. WritePreparedTransactionTest can use the constructor directly.